index.html

```diff
@@ -0,0 +1,139 @@
+                <img id="logoimglink" src="./images/CCC_logo-removebg-preview.png">
+            </a>
+            <nav id="desktopmenu">
+                <a class="menuitem" href="https://asdt560.github.io/Capstone-HTML-CSS-Basic-JS/index.html">Home</a>
+                <a class="menuitem" href="#">Program</a>
+                <a class="menuitem" href="#">Join</a>
+                <a class="menuitem" href="#">Sponsor</a>
+                <a class="menuitem" href="#">News</a>
+            </nav>
+        </section>
+        <section id="headline">
+           <h1> <p id="welcome">"Hello, creators!"</p></h1>
+            <h2 id="title">CONTENT CREATORS <br>
+                CONVENTION <br>
+                2022</h2>
+            <p id="openingblurb">An opportunity for creators from across the world to meet and discuss their work, this year hosted in Argentina</p>
+            <p id="date">2022.10.26(WED) ~ 16(FRI)</p>
+            <p id="place">@ Buenos Aires Convention Centre</p>
+        </section>
+    </header>
+    <main>
+        <section id="program">
+            <div class="titlecontainer">
+                <h2 class="sectiontitle program">Main Program</h2>
+                <span class="titleunderline"></span>
+            </div>
+            <div id="programholder">
+                <div class="programdiv">
+                    <img class="programimg" src="./images/lecture.png">
+                    <h3 class="programtitle">Lecture</h3>
+                    <p class="programblurb">Listen toe xperienced individuals share their knowledge on a variety of subjects</p>
+                </div>
+                <div class="programdiv">
+                    <img class="programimg" src="./images/exhibition.png">
+                    <h3 class="programtitle">Exhibition</h3>
+                    <p class="programblurb">Watch aspiring novices and veterans both show sneak-peeks of their work</p>
+                </div>
+                <div class="programdiv">
+                    <img class="programimg" src="./images/forum.png">
+                    <h3 class="programtitle">Forum</h3>
+                    <p class="programblurb">Share your thoughts and discuss a variety of subjects with experts on the topic</p>
+                </div>
+                <div class="programdiv">
+                    <img class="programimg" src="./images/workshop.png">
+                    <h3 class="programtitle">Workshop</h3>
+                    <p class="programblurb">Try your hand at creating something of your own rather than just watching</p>
+                </div>
+                <div class="programdiv">
+                    <img class="programimg" src="./images/ignite.png">
+                    <h3 class="programtitle">CCC Ignite</h3>
+                    <p class="programblurb">Get opportunities to network and build connections across the world</p>
+                </div>
+            </div>
```

Comment:
- you did here a great job, your main program section looks nice but kindly make sure you  add the hove effect to the card program which adding the `white border` as was required by the previous review  [here](https://github.com/elad237/capstone-project/pull/1#discussion_r1007175362)  :+1: